src/common/utility.ts

```diff
@@ -281,7 +281,8 @@ export const redirectToTradingPlatform = () =>
 
 // Function to manually add external js files.
 type TSettings = {
-    src: 'https://static.deriv.com/scripts/cookie.js'
+    src?: 'https://static.deriv.com/scripts/cookie.js'
+    text?: string
```

Comment:
do we need this?

Reply:
I guess yes, because they are passing a text to it in one of the files